### PR TITLE
Pytest

### PR DESCRIPTION
**Pull request by amjith**

### CODE REVIEW

**Comment by amjith** [comment]
I had to remove Python 3.8 since one of the advice in our sample uses `ast.unparse()`  which is not available in 3.8

File: setup.cfg [hunk 1]
```diff
@@ -35,6 +35,7 @@ dev =
     usort == 1.0.7
     wheel == 0.42.0
 test =
+    pytest
```
Comment (thatch) [inline setup.cfg:38]: at some point I'm going to add a min bound to this, but it's fine for now

File: Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@ setup:
 
 .PHONY: test
 test:
-	python -m coverage run -m advice_animal.tests $(TESTOPTS)
+	python -m coverage run -m pytest $(TESTOPTS)
```
Comment (thatch) [inline Makefile:19]: is this the recommended way, or to use `pytest-cov`?

**Review by thatch — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (amjith) [reply Makefile:19]: I think both ways are fine but `pytest-cov` requires an additional dependency. 🤷

**Pull request merged**